base/cypress.md

```diff
@@ -0,0 +1,41 @@
+# Cypress
+
+Cypress is a next generation front end testing tool built for the modern web. Complete documentation can be found [here](https://docs.cypress.io/guides/overview/why-cypress.html#In-a-nutshell).
+
+This tool is integrated to RERO-ILS and the following documentation explains how to use it.
+
+## Prerequisites
+
+`poetry run bootstrap` and `poetry run setup`
+Local server (localhost:5000) must be running and the database must contain data, as Cypress will use them to run the tests.
+
+## Configuration
+
+`poetry run cypress` will install Cypress if it's not present and launch the tests in command line.
+
+## Run the tests
+
+The tests can be ran on command line or with a graphic test launcher. The second option allows to see the tests running, see the snapshots for each step and launch them individually.
+
+### Command line
+
+`poetry run cypress`
+A video of the tests is created in folder tests/e2e/cypress/cypress/videos.
+
+### With the test launcher
+
+* `cd tests/e2e/cypress`
+* `npm run cypress` to open Cypress
+* Run the test by clicking on corresponding `spec.js` file
+
+## Writing the tests
+
+### Important folders
+
+The base url is stored in cypress.json file
+
+Re-usable code is written here: tests/e2e/cypress/cypress/support/commands.js
```

Comment:
Is there any Cyrpess documentation link about this kind of file? (to have more info about that)

Reply:
Done